docs/02_cva6_requirements/cva6_requirements_specification.rst

```diff
@@ -311,6 +311,11 @@ independent requirements.
 |                                   | **Zbc** and **Zbs** extensions    |
 |                                   | (bit manipulation), version 1.0.  |
 +-----------------------------------+-----------------------------------+
+| ISA-130                           | CVA6 should support as an         |
+|                                   | **option** the **Zicond**         |
+|                                   | extension, version 1.0.           |
```

Comment:
I thought that Zicond was not ratified yet. May I suggest to reword "CVA6 should support as an **option** the **Zicond** extension (ratification pending).